tests/Functional/FunctionalTest.php

```diff
@@ -40,6 +41,9 @@ class FunctionalTest extends ApplicationTestCase
 
     protected function setUp(): void
     {
+        self::prepareContainer(new Configuration([
+            __DIR__ . '/../../packages/guides/resources/template/html/guides',
```

Comment:
Should we not use the default config here, to ensure the default is working as expected. So not passing this array